addons/link_tracker/models/mail_render_mixin.py

```diff
@@ -38,6 +38,9 @@ def _shorten_links(self, html, link_tracker_vals, blacklist=None, base_url=None)
         short_schema = base_url + '/r/'
         for match in set(re.findall(tools.HTML_TAG_URL_REGEX, html)):
             long_url = match[1]
+            # Make relative links absolute
+            if long_url.startswith(('/', '?', '#')):
+                long_url = base_url + long_url
```

Comment:
and I guess technically urls.url_join is better, since it's already imported anyway apparently

Reply:
+1 for url_join but the other places in the code are doing a string concatenation already, I decided to be consistent
we discussed at work about changing every other place in a dedicated commit to use url_join everywhere, our conclusion was "meh in stable" and "meh doing a PR in master just for that"